torch/csrc/distributed/c10d/Ops.cpp

```diff
@@ -118,6 +118,26 @@ c10::intrusive_ptr<ProcessGroup::Work> barrier(
       BarrierOptions{device_ids, std::chrono::milliseconds(timeout)});
 }
 
+c10::intrusive_ptr<ProcessGroup::Work> send(
+    at::TensorList tensors,
```

Comment:
same comment on `TensorList` vs `std::vector<Tensor>`

Reply:
I think it's a convention to use TensorList to represent const std::vectorat::Tensor&.

I only use const std::vectorat::Tensor& if there is a const std::vector<std::vectorat::Tensor>& in the signature to keep consistency on that function. Let me know which way you like.